bucket.go

```diff
@@ -0,0 +1,70 @@
+package reductgo
```

Comment:
at some point we will generate API reference, please, add doc strings fro all public methods